charts/stash/templates/cluster-role.yaml

```diff
@@ -120,3 +121,8 @@ rules:
     - volumesnapshotcontents
     - volumesnapshotclasses
   verbs: ["create", "get", "list", "watch", "patch"]
```

Comment:
@jnaulty Can you please add `delete` verb here. It's also missing.
https://github.com/stashed/installer/blob/c0ff64759056fee0fccde8d8b93f4dcf46fb5b22/deploy/rbac-list.yaml#L120

Reply:
done